mingw-w64-git/mingw-w64-git.mak

```diff
@@ -106,4 +106,4 @@ install-mingit-busybox-test-artifacts:
 	# git-difftool--helper for t7800
 	install -m755 -d '$(DESTDIR_SQ)/$(MINGW_PREFIX)/libexec/git-core'
 	install -m755 git-difftool--helper \
-		'$(DESTDIR_SQ)/$(MINGW_PREFIX)/libexec/git-core'
\ No newline at end of file
+		'$(DESTDIR_SQ)/$(MINGW_PREFIX)/libexec/git-core'
```

Comment:
Please drop this diff hunk from the PR. It does not introduce any functional change, and therefore I would like to keep it out.